src/NUnitFramework/framework/Internal/OSPlatform.cs

```diff
@@ -228,6 +229,37 @@ public PlatformID Platform
         }
 
         /// <summary>
+        /// Return a 
+        /// </summary>
+        /// <returns></returns>
+        public override string ToString()
```

Comment:
The summary-element seems a bit incomplete. Also I would leave out the returns-element if it is empty.

Reply:
Yeah...not sure how that happened.  Thanks!